src/psfmachine/machine.py

```diff
@@ -749,7 +857,7 @@ def plot_time_model(self):
         cbar.set_label("Normalized Flux")
         return fig
 
-    def build_shape_model(self, plot=False, flux_cut_off=1):
+    def build_shape_model(self, plot=False, flux_cut_off=1, **kwargs):
```

Comment:
add `kwargs` so we can pass custom parameters to `_get_source_mask` if necessary.

Reply:
Is it better to have a specific keyword? That way we can have it in a docstring.